assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -354,7 +354,11 @@ che.infra.kubernetes.ingress.domain=
 che.infra.kubernetes.namespace=
 
 # Defines Kubernetes Service Account name which should be specified to be bound to all workspaces pods.
-# Note that Che Server won't create the service account and it should exist.
+# Note that Kubernetes Infrastructure won't create the service account and it should exist.
+# OpenShift infrastructure will check if project is predefined:
```

Comment:
when you say "if project is predefined" you mean "if `che.infra.openshift.project` is set" right?

Reply:
Right, I'll add the corresponding comment here.